### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_SchemaCompareOptions_SchemaCompare_Screenreader: NVDA doesn't announce Title of the dialog when opened.

### ISSUE THREAD

**Issue by ShwetaBaglane** [microsoft/azuredatastudio#6728]
[“Check out Accessibility Insights! !](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.10.0-insider (user setup)
Commit: 975e45061b073ecce2398d00c12f525d90e07b02
Date: 2019-08-08T19:22:13.572Z
VS Code 1.37.0
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
Nvda Version: 2019.2rc2

**Additional Details:**
Tools Name: Nvda
MAS References: MAS2.4.2

**Repro Steps:**

1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Navigate till Select Source or Select Target & hit enter.
9. Provide all the details for Source & Target.
10. Navigate till Ok button & hit enter.
11. Start Screen reader.
11. Navigate till Options menu from Schema compare Menu bar & hit enter.

**Actual:**
When clicked on 'Options' button, Schema compare options dialog opens but NVDA doesn't announce Title of the dialog.

**Expected:**
NVDA should announce Title of the dialog when Schema compare options dialog opens.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: (https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
Screen reader only users will face difficulty as they will not get to know the context of opened dialog.

**MAS Reference:**
MAS 2.4.2-
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={c74ff881-82b6-4401-92cf-8919f9bffaf2}

**Attachment for Reference:**
[A11y_AzureDataStudio(Schema,Dacpac,Backup)_SchemaCompareOptions_SchemaCompare_Screenreader_DialogTitle.pptx](https://github.com/microsoft/azuredatastudio/files/3496901/A11y_AzureDataStudio.Schema.Dacpac.Backup._SchemaCompareOptions_SchemaCompare_Screenreader_DialogTitle.pptx)

**Comment by fsteffi** [comment]
The issue has been solved but the control type after the announcement of the dialog is announced as document, which is not required. It should just announce the name of the dialog opened and then the tab selected, the unnecessary control as document should not be announced.
[#6728A11y_AzureDataStudio(Schema,Dacpac,Backup)_SchemaCompareOptions_SchemaCompare_Screenreader.pptx](https://github.com/microsoft/azuredatastudio/files/3712375/6728A11y_AzureDataStudio.Schema.Dacpac.Backup._SchemaCompareOptions_SchemaCompare_Screenreader.pptx)

**Comment by kisantia** [comment]
Removed the unnecessary announcing of document for dialogs
